src/store/accounts/index.ts

```diff
@@ -154,11 +154,12 @@ export const currentAccountDataState = atom(get => {
 
 export const currentAccountAvailableStxBalanceState = atom(get => {
   const currentAccountData = get(currentAccountDataState);
+  const stxBalance = get(accountInfoState)?.balance.toString(10);
   if (!currentAccountData) return;
   const stx = new BigNumber(currentAccountData.balances.stx.balance);
   const lockedStx = new BigNumber(currentAccountData.balances.stx.locked);
   const curBalance = stx.minus(lockedStx);
-  return curBalance;
+  return (stxBalance && new BigNumber(stxBalance)) || curBalance;
```

Comment:
@fbwoolf shouldn't this be the inverse?
```suggestion
  return curBalance || (stxBalance && new BigNumber(stxBalance));
```